core/src/main/resources/hudson/model/AbstractModelObject/descriptionForm.jelly

```diff
@@ -40,15 +40,14 @@ THE SOFTWARE.
   <j:set var="initialDescription" value="${request2.getParameter('description')}" defaultValue="${it.description}"/>
   <l:ajax>
     <form action="${submissionUrl}" method="post">
-      <table>
-        <f:entry help="${app.markupFormatter.helpUrl}">
-          <f:textarea name="description" value="${initialDescription}"
-                      codemirror-mode="${app.markupFormatter.codeMirrorMode}" codemirror-config="${app.markupFormatter.codeMirrorConfig}" previewEndpoint="/markupFormatter/previewDescription"/>
-        </f:entry>
-      </table>
-      <div align="right">
+      <f:entry help="${app.markupFormatter.helpUrl}">
+        <f:textarea name="description" value="${initialDescription}"
+                    codemirror-mode="${app.markupFormatter.codeMirrorMode}" codemirror-config="${app.markupFormatter.codeMirrorConfig}" previewEndpoint="/markupFormatter/previewDescription"/>
+      </f:entry>
+      <f:bottomButtonBar>
```

Comment:
I think we could live without the `<f:bottomButtonBar> and just use a `div` with `jenkins-buttons-row jenkins-buttons-row--equal-width` - unsure if the stickyness/extra padding is helping.

Reply:
Indeed, that is better, the extra padding os not needed.

Reply:
Except for views, but there we already miss some margin between the edit description button and the Views tabbar
![image](https://github.com/user-attachments/assets/331cea8e-0f64-4aac-8b70-c9a9790e946c)
![image](https://github.com/user-attachments/assets/57da4239-032d-4a4a-a8d5-11b4eeb1287f)